common/tree/script.go

```diff
@@ -68,12 +69,21 @@ type CLTVMultisigClosure struct {
 	Locktime common.Locktime
 }
 
+// ConditionMultisigClosure is a closure that contains a condition and a
+// multisig closure. The condition is a boolean script that is executed with the
+// multisig witness.
+type ConditionMultisigClosure struct {
+	MultisigClosure
```

Comment:
shouldn't it be extension of `CLTVMultisigClosure`, which is itself an extension of `MultisigClosure`?